### PR TITLE
Adding issue triage section to the about section of the circuitverse website

### LINKED ISSUE #4963: In the about page, add a new section called Issues Triaging team

### PR DESCRIPTION
**Pull request by Pixelz123**
Fixes #4963 

### 
github_pr1_screenshot

 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new "Issues Triaging Team" section on the About page, showcasing team members with their names and images linked to their profiles.
	- Added descriptions to enhance user understanding of the team's role in managing issues.

- **Documentation**
	- Updated English locale file to include new keys for the "Issues Triaging Team" and its description.

### CODE REVIEW

**Review by Pixelz123 — COMMENTED** [review]
resolved long line flags

File: app/controllers/about_controller.rb [hunk 1]
```diff
@@ -60,5 +60,20 @@ def index
...
+      { name: "Pulkit Gupta",
+        img: "https://avatars.githubusercontent.com/u/76155456?v=4", link: "https://github.com/pulkit-30" },
+      { name: "Vaibhav Upreti",
+        img: "https://avatars.githubusercontent.com/u/85568177?v=4", link: "https://github.com/VaibhavUpreti" },
+      { name: "Prerna Sharma",
+        img: "https://avatars.githubusercontent.com/u/89515816?v=4", link: "https://github.com/Prerna-0202" },
+      { name: "Tanmoy Sarkar",
+        img: "https://avatars.githubusercontent.com/u/57363826?v=4", link: "https://github.com/tanmoysrt" },
+      { name: "Arnabdaz",
+        img: "https://avatars.githubusercontent.com/u/96580571?v=4", link: "https://github.com/Arnabdaz" }
```
Comment (tachyons) [inline app/controllers/about_controller.rb:76]:
```suggestion
      
```

**Comment by Pixelz123** [comment]
Thank you so much... its was my first ever PR

**Comment by Pixelz123** [comment]
Sure

**Pull request merged**